admin/src/pages/Add.jsx

```
@@ -0,0 +1,301 @@
+      formData.append("category", category);
+      formData.append("subCategory", subCategory);
+      formData.append("sizes", JSON.stringify(sizes));
+      formData.append("bestSeller", bestSeller);
+
+      image1 && formData.append("image1", image1);
+      image2 && formData.append("image2", image2);
+      image3 && formData.append("image3", image3);
+      image4 && formData.append("image4", image4);
+
+      // console.log(formData);
+      const res = await axios.post(backendUrl + "/api/product/add", formData, {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)